test/testcommon/src/main/java/io/pravega/test/common/TestUtils.java

```diff
@@ -48,4 +53,87 @@ public static int getAvailableListenPort() {
         throw new IllegalStateException(
                 String.format("Could not assign port in range %d - %d", BASE_PORT, MAX_PORT_COUNT + BASE_PORT));
     }
+
+    public static void testBlocking(RunnableWithException blockingFunction, Runnable unblocker) {
```

Comment:
I would really like some javadoc on this method. It looks useful, but it's not clear to me how to use it should I need something like this.

Also would this method not be more suited to live in `AssertExtensions`, since that's where we put all the verification methods? `TestUtils` has common utility methods, but nothing that does verification.

Reply:
done